provision.sh

```diff
@@ -177,24 +177,28 @@ if [ ! $SKIP_REQUIREMENTS ] ; then
       exit 1
   else
     # Install virtualenv
-    which -a virtualenv >> /dev/null
+    which -a pipenv >> /dev/null
     if [[ $? != 0 ]] ; then
-      pip install virtualenv
+      pip install pipenv
```

Comment:
@tharna This has the potential of "accidentally" installing pipenv to system python on macOS. In a perfect world devs should not mess with their system python, but instead install a sandboxed python with maybe a tool like pyenv. This is not a reason to reject this PR though, since this was already the case with the previous virtualenv installation.